academic-services/project-academic-knowledge/extended-attribute-deprecation-migration.md

```diff
@@ -0,0 +1,33 @@
+# Deprecation of Entity Extended Attributes
```

Comment:
Needs header, e.g.:
---
title: Removal of Extended entity attribute
description: Information about the removal of the Extended entity attribute and guidance for migration
ms.topic: reference
ms.date: 7/1/2020
---